great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -37,6 +42,96 @@ def __init__(
             data_context=data_context,
         )
 
+    def _plot_descriptive(self, line: alt.Chart):
+        descriptive_chart: alt.Chart = line
+        return descriptive_chart
+
+    def _plot_prescriptive(
+        self,
+        df: pd.DataFrame,
+        chart_title: str,
+        metric_label: str,
+        metric_type: str,
+        x_axis_label: str,
+        x_axis_type: str,
+        line: alt.Chart,
+        expectation_configurations: List[ExpectationConfiguration],
+    ):
+        for expectation_configuration in expectation_configurations:
+            if (
+                expectation_configuration.expectation_type
+                == "expect_table_row_count_to_be_between"
+            ):
+                min_value: float = expectation_configuration.kwargs["min_value"]
+                max_value: float = expectation_configuration.kwargs["max_value"]
+                prescriptive_chart: alt.Chart = (
+                    self.get_expect_domain_values_to_be_between_chart(
+                        self,
+                        df=df,
+                        chart_title=chart_title,
+                        metric_label=metric_label,
+                        metric_type=metric_type,
+                        x_axis_label=x_axis_label,
+                        x_axis_type=x_axis_type,
+                        line=line,
+                        min_value=min_value,
+                        max_value=max_value,
+                    )
+                )
+                return prescriptive_chart
+
+    def _plot(
+        self,
+        metric_names: List[str],
+        data: np.ndarray,
+        prescriptive: bool,
+        expectation_configurations: list[ExpectationConfiguration],
```

Comment:
@NathanFarmer Typo -- this probably should be `List` (capitalized).  Thanks!

Reply:
🙇🏻

Reply:
✅